Lib/argparse.py

```diff
@@ -1581,11 +1580,8 @@ def _get_optional_kwargs(self, *args, **kwargs):
         for option_string in args:
             # error on strings that don't start with an appropriate prefix
             if not option_string[0] in self.prefix_chars:
-                args = {'option': option_string,
-                        'prefix_chars': self.prefix_chars}
-                msg = _('invalid option string %(option)r: '
-                        'must start with a character %(prefix_chars)r')
-                raise ValueError(msg % args)
+                raise ValueError(f'invalid option string {option_string!r}: '
+                       f'must start with a character {self.prefix_chars!r}')
```

Comment:
```suggestion
                raise ValueError(f'invalid option string {option_string!r}: '
                                 f'must start with a character {self.prefix_chars!r}')
```

Reply:
This will be too long line.

Reply:
Ah, indeed! Though PEP8 says the arguments should be aligned. Maybe you could do this?

```suggestion
                raise ValueError(
                    f'invalid option string {option_string!r}: '
                    f'must start with a character {self.prefix_chars!r}')
```